### DISCUSSION TITLE
Relations displayed in a screen

### DISCUSSION THREAD

**Discussion by specialmoe** [Budibase/budibase#5581] [Help]
Hey,

I'm new to Budibase and I'm trying to find a way to show relationsships in one screen:

I have multible tables wich are related to (manly) one main table. I would like to build a screen where I kann update all tables at once. So I don't have only a dropdown field where I can select the related data from a different table, I would like to have the related data shown in the same screen with fields where I can edit also the rows wich are not the "display row" of the table.

for example:

I have a "VFX Shot" table and a "Turnover VFX Shot" table. When I'm viewig a VFX Shot record, I would like to see the "Turnover VFX Shot" coulm turnover_number and turnover_date. And if I change the turnover_number and turnover_date, it will make a new record in the "Turnover VFX Shot" (if it couldn't be a new record, it should update the one it's showed up, at least).

Would be nice if someone can helf me out :)

Best regards

**Comment by melohagan** [comment]
Hey @moritzpoth 

These should be of use:
- https://docs.budibase.com/docs/chaining-actions#tutorial-creating-and-linking-two-table-entries-in-one-form
- https://docs.budibase.com/docs/formula#fields-from-relationships
- https://docs.budibase.com/docs/save-two-linked-tables-in-one-form
- https://docs.budibase.com/docs/filter-by-relationships#tutorial-filter-on-selected-relationship-picker-values

Please let me know how you get on, and if there's anything in the docs that wasn't clear or can be improved upon.

**Reply by Solarbotics** [reply]
That first link 404'd. I think it's supposed to be [this?](https://github.com/Budibase/budibase/discussions/8087)

**Reply by melohagan** [reply]
Thanks @Solarbotics I've updated the docs link to be correct, and added an extra couple of links for good measure.
